README.md

```diff
@@ -1,2 +1,48 @@
-# tv-show
-This is a JavaScript capstone project about building a web application based on an external API. We selected an API, TV-Maze, that provides data about tv shows and then built the webapp around it. 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TV-Show Web App
+
+This is a JavaScript capstone project about building a web application based on an external API. We selected an API, TV-Maze, that provides data about tv shows and then built the webapp around it.
+
+## Built With
+
+- Major languages - HTML, SCSS & JS
+- Technologies used - VSCode, Webpack
+
+## Live Demo
+
+[Live Demo Link](https://walterokumu.github.io/tv-show/dist)
+
+## Getting Started
+
```

Comment:
Please make sure that you have recorded a presentation video presenting the features of the project with a maximum length of 5 minutes. The video must contain a clear and objective description of the project and highlights a piece of the code the student finds interesting or is proud of making.

**Note:** It's recommended that students use [Zoom](https://zoom.us/) recording features while in a call with their peers.

Reply:
Thanks; We recorded the video and is added to the README file